config.json

```diff
@@ -53,6 +53,10 @@
       "domain": "lyon.caresteouvert.fr",
       "defaultLocale": "fr",
       "disclaimer": "https://blog.caresteouvert.fr/mentions-legales-lyon"
+    },
+    "permaneceaberto": {
```

Comment:
There is no such brand defined in the assets (at least for now), no need to include it

Reply:
Out of curiosity, how would I edit the original logo to create a new brand?

Reply:
This can be done by our graphic designer once name in your language/country is chosen. Otherwise by editing SVG version of an existing logo.